yarn.lock

```diff
@@ -5382,7 +5387,7 @@ istanbul-reports@^1.5.1:
   dependencies:
     handlebars "^4.0.3"
 
-jaaslib@0.5.0, jaaslib@^0.5.0:
+jaaslib@^0.5.0:
```

Comment:
This would suggest something still depends on plain old `jaaslib`, no?

Reply:
yessir the juju-react-components lib does. It's on deck to get an upgrade :D 

Good catch 😉